scene/gui/popup_menu.cpp

```diff
@@ -1249,6 +1251,15 @@ void PopupMenu::set_item_icon_max_width(int p_idx, int p_width) {
 	_menu_changed();
 }
 
+void PopupMenu::set_item_icon_modulate(int p_idx, const Color &p_modulate) {
+	if (items[p_idx].icon_modulate == p_modulate) {
```

Comment:
You need to validate the index like in all other PopupMenu setters:
```
if (p_idx < 0) {
	p_idx += get_item_count();
}
ERR_FAIL_INDEX(p_idx, items.size());
```

The CI caught a crash here:
```
GDSCRIPT CODE:     PopupMenu.new().set_item_icon_modulate(100, Color(100, 100, 100, 1))
ERROR: FATAL: Index p_index = 100 is out of bounds (size() = 0).
   at: get (./core/templates/cowdata.h:155)
```

Reply:
Oops, didn't notice during the rebase, thanks!